src/alpine/3.9/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.9
+FROM python:3.7-alpine3.9
```

Comment:
It is beneficial to keep alignment between the Dockerfiles for the various architectures.  By this I mean the base images should align between this Dockerfile and the [corresponding arm Dockerfile](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/245/files#diff-81ce923e0871c5773a95153cf8e42270).